### ISSUE TITLE
Worse robustness than TRADES on Cifar-10 under AutoAttack

### ISSUE THREAD

**Issue by Equationliu** [ShawnXYang/AT_HE#1]
I compared the robustness of the released official ckpts under the [autoattack](https://github.com/fra31/auto-attack).

|                             Ckpt                             | Nature | After APGD-CE | After APGD-DLR |
| :----------------------------------------------------------: | :----: | :-----------: | :------------: |
|  [model_cifar_wrn.pt](https://github.com/yaodongyu/TRADES)   | 84.92% |    55.12%     |     53.34%     |
| [model-wideres-tradesHE.pt](https://github.com/ShawnXYang/AT_HE/tree/master/CIFAR-10) | 84.42% |    60.01%     |     52.91%     |

I guess why the `HE` method is not robust enough to the `APGD-DLR` loss is the logits range. As your paper mentioned, using adaptive attack can decrese the robustness, too.

**Comment by P2333** [comment]
Yeah, we also find that HE does not work well for TRADES. But HE is more suitable for PGD-AT, you can evaluate the PGD-AT+HE and see the results

**Comment by Equationliu** [comment]
> Yeah, we also find that HE does not work well for TRADES. But HE is more suitable for PGD-AT, you can evaluate the PGD-AT+HE and see the results

**Comment by Equationliu** [comment]
> Yeah, we also find that HE does not work well for TRADES. But HE is more suitable for PGD-AT, you can evaluate the PGD-AT+HE and see the results

I get the following result. 
|                             Ckpt                             | Nature | After APGD-CE | After APGD-DLR |
| :----------------------------------------------------------: | :----: | :-----------: | :------------: |
| [model-wideres-pgdHE.pt](https://github.com/ShawnXYang/AT_HE/tree/master/CIFAR-10) | 86.19% |    56.89%     |     52.61%     |

**Comment by P2333** [comment]
Thank you for your report! 

Since our code is mostly based on the code of TRADES, we are re-training the model to see if we can match the performance of the public checkpoint of TRADES. Our re-implemented TRADES baseline has worse robustness (worse than TRADES+HE), so I wonder if there are some implicit tricks beyond the public TRADES code.

**Comment by P2333** [comment]
@Equationliu Hi, we have updated the checkpoints and codes with better performance :)
